recipes/repeated-test/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "repeated_test" %}
```

Comment:
Sorry, missed this earlier. Appears the directory and package name don't match. Should it be with a `-` or an `_`?